server/src/main/java/org/opensearch/index/engine/Engine.java

```diff
@@ -828,7 +828,7 @@ protected final void ensureOpen(Exception suppressed) {
         }
     }
 
-    protected final void ensureOpen() {
+    public final void ensureOpen() {
```

Comment:
@satyajitg28 this is a breaking change: the change in visibly breaks compilation, it is not reverted